src/unity/schemas/Account.yml

```diff
@@ -1,20 +1,9 @@
 properties:
   id:
-    type: string
-    description: account id
-  quartzId:
     type: number
     description: account id in quartz
-  onboardingState:
-    type: string
-    description: stage of onboarding the account is in
-  marketplace:
```

Comment:
making a distinction between marketplace (info about a marketplace) and a marketplace subscription (info about a user / accounts subscription to a marketlpace)